Cargo.toml

```
@@ -13,6 +13,8 @@ readme = "README.md"
 [dependencies]
 anyhow = "1.0.38"
 atty = "0.2"
+cookie_crate = { version = "0.15", package = "cookie" }
+cookie_store = "0.13"
```

cookie_store depends on reqwest which in turn enables `native-tls` 😕

---

Setting `default-features = false` in `cookie_store`'s `Cargo.toml` does the trick. Should I make a PR?

---

Yeah, I think that could work, but we would have to reimplement the `reqwest::cookie::CookieStore` trait for `cookie_str`, right?

>Should I make a PR?

Yes, please

---

If we set `default-features = false` for `cookie_store` in our `Cargo.toml` then we have to implement it ourselves, yeah.

You can't implement another crate's trait for another crate's type without a wrapper, but `CookieStoreMutex` is a wrapper anyway so I think we could just copy it over wholesale.

> Yes, please

Done: pfernie/cookie_store#15

For now this works:
```suggestion
cookie_store = { git = "https://github.com/blyxxyz/cookie_store", branch = "no-reqwest-default-features" }
```
It should be replaced before the next release, either by a new upstream release with the fix or by the manual implementation workaround.

---

Thank you very much!